app/api/v1/common/type_mapper.py

```diff
@@ -0,0 +1,38 @@
+from electionguard.group import (
+    ElementModP,
+    ElementModQ,
+    hex_to_p,
+    hex_to_q,
+    int_to_p,
+    int_to_q,
+)
+
+
+class TypeMapper:
+    @staticmethod
+    def string_to_element_mod_p(value: str) -> ElementModP:
+        elementmodp = hex_to_p(value)
+        if elementmodp is None:
+            raise ValueError("invalid key: " + value)
+        return elementmodp
+
+    @staticmethod
+    def string_to_element_mod_q(value: str) -> ElementModQ:
+        elementmodq = hex_to_q(value)
+        if elementmodq is None:
+            raise ValueError("invalid key: " + value)
+        return elementmodq
+
+    @staticmethod
+    def int_to_element_mod_p(value: int) -> ElementModP:
```

Comment:
I would have preferred combining these and allowing both strings and integers on the DTO for flexibility on the API. If only so we can test with actual numbers we can read as opposed to exclusively hex. There is an incredible value to this.

If we're not doing `Union[str,int]` on that class and future `ElementModQ` or `ElementModP` situations, we do not need both int and string methods. That being said, the future code in electionguard-python 1.4 does support either.

Example:
```py
def convert_to_element_mod_p(value: Union[int,str]) -> ElementModP:
     """Convert to ElementModP"""

     element =  int_to_p(value) if isinstance(value, int) else hex_to_p(value)
     if element is None:
            raise ValueError(type_value_message(value, ElementModP))
     return element
```